src/NzbDrone.Common/Exceptions/DestinationAlreadyExistsIOException.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.IO;
+using System.Runtime.Serialization;
+
+namespace NzbDrone.Common
+{
+    public class DestinationAlreadyExistsIOException : IOException
```

Comment:
One small nitpick, could you rename this to `DestinationAlreadyExistsException`, otherwise I think it's good.